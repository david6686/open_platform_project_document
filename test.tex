\documentclass[12pt,a4paper]{article}
\usepackage{xltxtra,fontspec,xunicode}
\usepackage[slantfont,boldfont]{xeCJK} % 允许斜体和粗体

\setCJKmainfont{Kai}   % 设置缺省中文字体
\setCJKmonofont{Hei}   % 设置等宽字体
\setmainfont{Optima}   % 英文衬线字体
\setmonofont{Monaco}   % 英文等宽字体
\setsansfont{Trebuchet MS} % 英文无衬线字体

\begin{document}
\begin{verse}
  Stray birds of summer come to my window to sing and fly away. \\
  And yellow leaves of autumn, which have no songs, \\
  flutter and fall there with a sign.\\
  \hfill \emph{Rabindranath Tagore}
\end{verse}

\begin{verse}
  夏天的飞鸟，飞到我的窗前唱歌，又飞去了。\\
  秋天的黄叶，它们没有什么可唱，只叹息一声，飞落在那里。\\
  \hfill \emph{罗宾德拉纳特·泰戈尔}
\end{verse}
\end{document}
