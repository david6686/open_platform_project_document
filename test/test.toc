\contentsline {section}{\numberline {1}購買連結}{1}{section.1}
\contentsline {subsection}{\numberline {1.1}\href {http://goods.ruten.com.tw/item/show?21607818417724url}{第一期}}{1}{subsection.1.1}
\contentsline {subsection}{\numberline {1.2}\href {http://goods.ruten.com.tw/item/show?21607818426948}{第二期} }{1}{subsection.1.2}
\contentsline {subsection}{\numberline {1.3}\href {http://goods.ruten.com.tw/item/show?21537612540837}{第三期} }{1}{subsection.1.3}
\contentsline {subsection}{\numberline {1.4}\href {http://goods.ruten.com.tw/item/show?21607807273777}{第四期} }{1}{subsection.1.4}
\contentsline {subsection}{\numberline {1.5}\href {http://goods.ruten.com.tw/item/show?21633348062558}{第五期} }{1}{subsection.1.5}
\contentsline {paragraph}{露天拍賣適用合併運費規則，一次買多本只須付一份運費。另外，也可以在奧丁丁網路開放市集購買全部：\url {https://www.owlting.com/market/items/1722}}{1}{section*.2}
\contentsline {paragraph}{另，動畫《秘封活動記錄～月～第一話》ＤＶＤ與前傳漫畫《月之追憶》的購買連結如下：url\{http://goods.ruten.com.tw/item/show?21607807078107}{2}{section*.3}
\contentsline {section}{\numberline {2}簡介}{2}{section.2}
\contentsline {paragraph}{《東方文化學刊》是一本同人誌，以《東方Project》系列作及其同人文化為主題，創刊於2015年6月的台北，主編為「恆萃工坊」創辦人胡又天。這部同人誌的書名和內文格式都仿照正規學術期刊，為的是「給大家打氣」：證明遊戲、動漫和其他藝術形式一樣，當得起認真的研究與品評；透過發掘《東方Project》各作的歷史、民俗內涵，探討其中音樂、美術、遊戲設計等方面的學問，我們可以讓廣大同好更深入瞭解東方的世界，也為創作者提供借鑑，去作出更加精緻的作品。}{2}{section*.4}
\contentsline {paragraph}{東方文化學刊》一、二期首發於2015年6月6日、在三重體育館舉辦的「博麗神社例大祭in 台灣」，第三期於9月出刊；2016年1月，前三期再版。第四期將首發於2016年2月21日、在台大綜合體育館舉辦的「Comic Horizon漫創地平線」活動中的「博麗神社例大祭 in台灣 1.5」區塊，攤位編號E13。}{2}{section*.5}
\contentsline {section}{\numberline {3}第一期 3　　同人于東方之野}{3}{section.3}
\contentsline {paragraph}{\bf 發刊詞：我們的心懷是博麗而永遠的}{3}{section*.6}
