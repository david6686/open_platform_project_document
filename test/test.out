\BOOKMARK [1][-]{section.1}{購買連結}{}% 1
\BOOKMARK [2][-]{subsection.1.1}{第一期}{section.1}% 2
\BOOKMARK [2][-]{subsection.1.2}{第二期 }{section.1}% 3
\BOOKMARK [2][-]{subsection.1.3}{第三期 }{section.1}% 4
\BOOKMARK [2][-]{subsection.1.4}{第四期 }{section.1}% 5
\BOOKMARK [2][-]{subsection.1.5}{第五期 }{section.1}% 6
\BOOKMARK [1][-]{section.2}{簡介}{}% 7
\BOOKMARK [1][-]{section.3}{第一期 3　　同人于東方之野}{}% 8
